api/api.go

```diff
@@ -73,11 +74,22 @@ func (a *API) ListenAndServe(hostAndPort string) error {
 	return http.ListenAndServe(hostAndPort, a.handler)
 }
 
+func timeRequest(ctx context.Context, w http.ResponseWriter, r *http.Request) context.Context {
+	return context.WithValue(ctx, "_netlify_commerce_timing", time.Now())
+}
+
+func logHandler(ctx context.Context, wp mutil.WriterProxy, req *http.Request) {
+	start := ctx.Value("_netlify_commerce_timing").(time.Time)
+	logrus.Infof("method=%s path=%s status=%v duration=%v", req.Method, req.URL.Path, wp.Status(), time.Since(start))
```

Comment:
On second thought, I like better the current format, I want that specific order, and logrus sorts alphabetically using Fields. Comparison:

Logrus:

```
INFO[0000] Netlify Commerce API started on: :8080       
INFO[0004]                                               duration=6.985µs method=GET path=/ status=200
INFO[0013]                                               duration=25.74µs method=GET path=/ status=200
```

Just Text:

```
INFO[0000] Netlify Commerce API started on: :8080       
INFO[0002] method=GET path=/ status=200 duration=10.025µs 
INFO[0005] method=GET path=/ status=200 duration=28.816µs 
```